staramr/blast/plasmidfinder/PlasmidfinderBlastDatabase.py

```diff
@@ -12,12 +14,52 @@
 
 class PlasmidfinderBlastDatabase(AbstractBlastDatabase):
 
-    def __init__(self, database_dir: str) -> None:
+    def __init__(self, database_dir: str, database_type: str = None) -> None:
         """
         Creates a new PlasmidfinderBlastDatabase.
-        :param database_dir: The specific ResFinder database (drug class) directory.
+        :param database_dir: The specific PlasmidFinder database directory.
         """
+
         super().__init__(database_dir)
+        self.plasmidfinder_database_dir = path.join(self.database_dir)
+        self.database_type = database_type
+
+        if database_type is not None:
+            self.plasmidfinder_database_file = path.join(self.database_dir, database_type)
+            self.plasmidfinder_database_file += self.fasta_suffix
+
+            if (not path.isfile(self.plasmidfinder_database_file)):
+                raise Exception("Error, plasmidfinder database type [{}] is either incorrect or plasmidfinder database not installed properly", database_type)
+            elif database_type not in PlasmidfinderBlastDatabase.get_database_types(database_dir):
+                raise Exception("Plasmidfinder database type [{}] is not valid", database_type)
 
     def get_name(self) -> str:
         return 'plasmidfinder'
+
+    def get_database_names(self):
```

Comment:
Could you add a return type here? You could likely make it `List[str]`

Reply:
Yup already did in 48f2a20